BUILDING.md

```diff
@@ -188,6 +199,12 @@ To run the tests:
 > .\vcbuild test
 ```
 
+To run the tests and generate coverage reports:
```

Comment:
Windows (i.e. `vcbuild`) does not have a counterpart, so you can remove this section

Reply:
Thanks for your review @refack I will update the doc.